spec/abstractops.html

```diff
@@ -651,40 +651,190 @@ <h1>
     </emu-alg>
   </emu-clause>
 
-  <emu-clause id="sec-temporal-roundhalfawayfromzero" aoid="RoundHalfAwayFromZero">
-    <h1>RoundHalfAwayFromZero ( _x_ )</h1>
+  <emu-clause id="sec-temporal-getunsignedroundingmode" type="abstract operation">
+    <h1>
+      GetUnsignedRoundingMode (
+        _roundingMode_: *"ceil"*, *"floor"*, *"expand"*, *"trunc"*, *"halfCeil"*, *"halfFloor"*, *"halfExpand"*, *"halfTrunc"*, or *"halfEven"*,
+        _isNegative_: *true* or *false*,
+      ): ~zero~, ~infinity~, ~half-zero~, ~half-infinity~, or ~half-even~
+    </h1>
+    <dl class="header">
+      <dt>description</dt>
+      <dd>
+        The return value is the rounding mode that should be applied to the absolute value of a number to produce the same result as if _roundingMode_ were applied to the signed value of the number (negative if _isNegative_ is *true*, or positive otherwise).
+      </dd>
+    </dl>
     <emu-alg>
-      1. Assert: _x_ is a mathematical value.
-      1. Return the mathematical value that is closest to _x_ and is an integer.
-         If two integers are equally close to _x_, then the result is the integer that is farther away from 0. If _x_ is already an integer, then the result is _x_.
+      1. If _isNegative_ is *true*, return the specification type in the third column of <emu-xref href="#table-temporal-unsigned-rounding-modes"></emu-xref> where the first column is _roundingMode_ and the second column is "negative".
+      1. Else, return the specification type in the third column of <emu-xref href="#table-temporal-unsigned-rounding-modes"></emu-xref> where the first column is _roundingMode_ and the second column is "positive".
     </emu-alg>
+    <emu-note type="editor">
+      <p>This operation is intended to be the same one as in the <a href="https://tc39.es/proposal-intl-numberformat-v3/out/numberformat/diff.html#sec-getunsignedroundingmode">Intl.NumberFormat v3</a> proposal.</p>
+    </emu-note>
+    <emu-table id="table-temporal-unsigned-rounding-modes">
+      <emu-caption>Conversion from rounding mode to unsigned rounding mode</emu-caption>
+      <table class="real-table">
+        <thead>
+          <tr>
+            <th>Identifier</th>
+            <th>Sign</th>
+            <th>Unsigned Rounding Mode</th>
+          </tr>
+        </thead>
```

Comment:
Since **Sign** is always "positive" or "negative", wouldn't an alternate arrangement be more clear?
> | Identifier  | Non-negative Rounding Mode | Negative Rounding Mode |
> | ------------- | ------------- | ------------- |
> | **"ceil"**  | \~infinity~  | \~zero~ |
> | **"floor"**  | \~zero~  | \~infinity~ |
> | … | … | … |

Reply:
This should probably be addressed in NFv3 as well? @sffc, what do you think?